docs/api/cases/cases-api-find-cases.asciidoc

```diff
@@ -0,0 +1,171 @@
+[[cases-api-find-cases]]
+== Find cases API
+++++
+<titleabbrev>Find cases</titleabbrev>
+++++
+
+Retrieves a paginated subset of cases. By default, the first page is returned
+with 20 results per page.
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/_find`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+=== Query parameters
+
+`owner`::
+(Optional, string or array of strings) A filter to limit the retrieved cases to a
+specific set of applications. Valid values are: `cases`, `observability`,
+and `securitySolution`. If this parameter is omitted, the response contains all
+cases that the user has access to read.
+
+`page`::
+(Optional, integer) The page number to return.
+
+`perPage`::
+(Optional, integer) The number of rules to return per page.
+
+`reporters`::
+(Optional, string) Filters the returned cases by the reporter's `username`.
+
+`search`::
+(Optional, string) An {es}
+{ref}/query-dsl-simple-query-string-query.html[simple_query_string] query that
+filters the objects in the response.
+
+`sortField`::
+(Optional, string) Determines which field is used to sort the results,
+`createdAt` or `updatedAt`.
++
+NOTE: Even though the JSON case object uses `created_at` and `updated_at`
+fields, you must use `createdAt` and `updatedAt` fields in the URL
+query.
+
+`sortOrder`::
+(Optional, string) Determines the sort order, which can be `desc` or `asc`.
+
+`status`::
+(Optional, string) Filters the returned cases by state, which can be  `open`,
+`in-progress`, or `closed`.
+
+`tags`::
+(Optional, string) Filters the returned cases by tags.
+
+=== Response code
+
+`200`::
+   Indicates a successful call.
+
+=== Example
+
+Retrieve the first five cases with the `phishing` tag, in ascending order by
+last update time:
+
+[source,sh]
+--------------------------------------------------
+GET api/cases/_find?page=1&perPage=5&sortField=updatedAt&sortOrder=asc&tags=phishing
+--------------------------------------------------
+// KIBANA
+
+The API returns a JSON object listing the retrieved cases. For example:
+
+[source,json]
+--------------------------------------------------
+{
+  "page": 1,
+  "per_page": 5,
+  "total": 2,
+  "cases": [
+    {
+      "id": "abed3a70-71bd-11ea-a0b2-c51ea50a58e2",
+      "version": "WzExMCwxXQ==",
+      "comments": [],
+      "totalComment": 0,
+      "totalAlerts": 0,
```

Comment:
>      "totalAlerts": 0,

This property wasn't in the original example in https://www.elastic.co/guide/en/security/master/cases-api-find-cases.html, so it was excluded intentionally, let me know!